code/game/objects/items/food/burgers.dm

```diff
@@ -624,7 +624,7 @@
 	venue_value = FOOD_PRICE_NORMAL
 	crafting_complexity = FOOD_COMPLEXITY_3
 
-/obj/item/food/burger/cheese
+/obj/item/food/burger/royale_with_cheese
```

Comment:
thank you for correcting the name to royale with cheese because as a european I am not familiar with the weight of cheese.